src/main/java/no/nav/tag/finnkandidatapi/tilretteleggingsbehov/Tilretteleggingsbehov.java

```diff
@@ -0,0 +1,23 @@
+package no.nav.tag.finnkandidatapi.tilretteleggingsbehov;
+
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+@Data
+@Builder
+public class Tilretteleggingsbehov {
+
+    @Id
+    private Integer id;
+    private LocalDateTime opprettet;
+    private String opprettetAvIdent;
+    private String brukerFnr;
+    private Arbeidstid arbeidstid;
+    private ArrayList<Fysisk> fysisk;
```

Comment:
Vanligst å her bruke interfacet `List` i stedet for `ArrayList`.

Reply:
Fordi man ønsker å bruke interface slik at man kan bestemme selv hvilken implementasjon man ønsker å bruke? [Her](https://github.com/navikt/finn-kandidat-api/pull/6/files#diff-83aa4b9ba519a336a7797acf1c8719b0R43) og [her](https://github.com/navikt/finn-kandidat-api/pull/6/files#diff-012870f9133a600b247815847033bdb3R24) bruker man fortsatt ArrayList?

Reply:
Glem siste spørsmål, fikk fjernet ArrayList.

Reply:
Egentlig hadde det vel vært enda mer ryddig å bruke Set i stedet for List: Rekkefølgen er vel uviktig (vi bør iallfall ikke være avhengig av at svarene persisteres i en viss rekkefølge) og hvert svar skal maks forekomme én gang.